code/controllers/subsystem/air.dm

```diff
@@ -391,7 +391,7 @@ SUBSYSTEM_DEF(air)
 		currentrun.len--
 		EG.breakdown_cooldown++
 		EG.dismantle_cooldown++
-		if(EG.breakdown_cooldown >= EXCITED_GROUP_BREAKDOWN_CYCLES)
+		if(EG.breakdown_cooldown >= EXCITED_GROUP_BREAKDOWN_CYCLES && !(EG.turf_reactions & (REACTING | VOLATILE_REACTION)))
```

Comment:
what do you intend this check to do? also I am not sure if this would lead to breakdown -> dismantle in one tick

Reply:
Fail if there is a reaction that is REACTING and is a VOLATILE_REACTION.